### PR TITLE
Move to Broccoli 1.0 and use the latest angular-cli

### PR DESCRIPTION
**Pull request by hansl**

### CODE REVIEW

**Comment by hansl** [comment]
This needs the next 0.0.28 release of angular-cli, which should happen today.

File: ember-cli-build.js [hunk 1]
```diff
@@ -70,16 +69,22 @@ function getDartTree(root) {
     formatter
   ]);
 
-  const pubSpecTree = new BroccoliFunnel(new BroccoliSource.UnwatchedDir('.'), {
+  // Almost the same as using a plain string for a tree; but unlike a plain
+  // string, Broccoli won't watch this
+  function UnwatchedTree (dir) { this.dir = dir }
+  UnwatchedTree.prototype.read = function (readTree) { return this.dir }
+  UnwatchedTree.prototype.cleanup = function () { }
```
Comment (jelbourn) [inline ember-cli-build.js]: Make this an es6 class?

**Comment by jelbourn** [comment]
LGTM

File: ember-cli-build.js (hunk 1 above)
Comment (hansl) [inline ember-cli-build.js]: good catch. done. retesting

**Comment by hansl** [comment]
Fixed the Broccoli plugins for TS2Dart and DartFormatter.  @jelbourn are you still okay with it?  I'll let you an hour to let me know :)

**Comment by jelbourn** [comment]
Still lgtm

**Comment by hansl** [comment]
This has been submitted as part of https://github.com/angular/material2/commit/6b7a35cb9f9c772c676a5cffc40ab5224a540116.

**Pull request closed without merging**